packages/client/src/views/PrintCertificate/PDFUtils.ts

```diff
@@ -143,7 +167,12 @@ export function downloadFile(
   data: string,
   fileName: string
 ) {
-  const linkSource = `data:${contentType};base64,${window.btoa(data)}`
+  let linkSource
+  if (data.includes('data:image/svg+xml;base64')) {
```

Comment:
maybe it would be better if we use `data:${contentType};base64`

Reply:
> maybe it would be better if we use `data:${contentType};base64`

I have addressed this.